src/main/sensors/battery.h

```diff
@@ -41,6 +41,8 @@
 #define MAX_BATTERY_PROFILE_COUNT SETTING_CONSTANT_MAX_BATTERY_PROFILE_COUNT
 #endif
 
+#define BATT_LOW_VOLT_TIMEOUT_MS 10000 // low voltage of 10 seconds
```

Comment:
@JulioCesarMatias I fear 10s is too long. In my opinion, 2-3s would be actually better for the pilot. What do you think?

Reply:
I still think 2-3 seconds is too little, how about 5 seconds? Because the intent is to make iNav 100% sure the battery is low before alerting the pilot.